build_runner/lib/src/package_graph/package_graph.dart

```diff
@@ -167,6 +171,7 @@ PackageDependencyType _dependencyType(source) {
       case 'hosted':
         return PackageDependencyType.hosted;
       case 'path':
+      case 'sdk': // Until Flutter supports another type, assume same as path.
```

Comment:
That is my understanding. Maybe @cbracken or @yjbanov or @devoncarew know better.